recipe/meta.yaml

```diff
@@ -11,30 +11,34 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python >=3.7
     - pip
     - pytest-runner
+    - setuptools_scm
   run:
     - python >=3.7
     - xarray
     - pandas
     - numpy
+    - importlib-metadata
```

Comment:
Only need for Python < 3.8 but doesn't hurt on other versions

Reply:
Can you actually add a comment indicating this? I think in the near future we might drop 3.8 and then it would be nice to have a reminder to remove this again?

Reply:
Done